### DISCUSSION TITLE
Support for Brotli Encoding

### DISCUSSION THREAD

**Discussion by pheaver** [AsyncHttpClient/async-http-client#1920] [Ideas]
Hello, would you consider a PR that adds support for Brotli encoding? I could implement it against version 2.8.1 or 2.12.3, whichever you would prefer.

Thanks!

**Comment by hyperxpro** [comment]
Hi

You can do PR against 3.0 on `main` branch.

**Reply by pheaver** [reply]
Thanks, I'd be happy to work on that and submit a PR against 3.0!

However, our company uses 2.x right now and I doubt we will upgrade anytime soon, so it would be really nice to also have this backported to 2.x. If that's not something you're willing to do, I totally understand, and we would likely just maintain a fork of our own until we're ready to switch to 3.x.
